code/game/machinery/floodlight.dm

```diff
@@ -1,22 +1,20 @@
-//these are probably broken
-
 /obj/machinery/floodlight
-	name = "Emergency Floodlight"
+	name = "emergency floodlight"
 	icon = 'icons/obj/machines/floodlight.dmi'
 	icon_state = "flood00"
-	anchored = 0
-	density = 1
-	var/on = 0
+	anchored = FALSE
+	density = TRUE
+	var/on = FALSE
 	var/obj/item/stock_parts/cell/high/cell = null
 	var/use = 5
-	var/unlocked = 0
-	var/open = 0
+	var/unlocked = FALSE
+	var/open = FALSE
 	var/brightness_on = 14
 	light_power = 20
-	//var/brightness_on = 999		//can't remember what the maxed out value is //Lighting overhaul: No max, stop TRYING TO ILLUMINATE MORE TILES THAN THE MAP CONSISTS OF.
 
 /obj/machinery/floodlight/New()
 	src.cell = new(src)
+	turnOn()
```

Comment:
This proc is kinda not well named. It implies it would turn on the floodlight, but it actually is just an init proc. I'd call this something more like `mapVarsInitialization` or something like that.

Also, it should turn on in `/Initialize(`, not `/New(`.

Reply:
All fair comments. Will improve.